src/pages/legal.tsx

```diff
@@ -0,0 +1,48 @@
+import { micromark } from 'micromark'
+import { GetStaticProps } from 'next'
+
+import { FrontendClientBase } from '@/components/frontend-client-base'
+import { renderedPageNoHooks } from '@/helper/rendered-page'
+
+export const legalRepo =
+  'https://raw.githubusercontent.com/serlo/serlo.org-legal/main'
+
+export interface LegalData {
+  contentHtml: string
+  isGerman: boolean
+}
+
+export default renderedPageNoHooks<LegalData>((props) => {
+  return <Content {...props} />
+})
+
+function Content({ contentHtml, isGerman }: LegalData) {
+  return (
+    <FrontendClientBase>
+      <h1 className="serlo-h1 mt-24">
+        {isGerman ? 'Impressum' : 'Legal Notice'}
+      </h1>
+      <main
+        className="serlo-prose-hacks"
+        dangerouslySetInnerHTML={{ __html: contentHtml }}
+      />
+    </FrontendClientBase>
+  )
+}
+
+export const getStaticProps: GetStaticProps<LegalData> = async (context) => {
+  const isGerman = context.locale === 'de'
+  const url = `${legalRepo}/${isGerman ? 'de' : 'en'}/imprint.md`
+
+  try {
+    const response = await fetch(url)
+    const markdown = await response.text()
+    const contentHtml = micromark(markdown, {
+      allowDangerousHtml: true,
```

Comment:
input markdown example:
```
Serlo Education e. V.<br>
IBAN: DE98 4306 0967 8204 5906 00<br>
BIC: GENODEM1GLS (GLS Bank)

Knr: 8204590600<br>
BLZ: 43060967
```

If we don't use the setting we end up with `<br>` in the output.
Options:
- make sure we never use any html in the markdown files
- add the [tagfilter plugin](https://github.com/micromark/micromark-extension-gfm-tagfilter)
- leave it as it is since the markdown is not user generated and the [source repo](https://github.com/serlo/serlo.org-legal/blob/main/de/imprint.md?plain=1) is as secure (or more so) as this repo

Reply:
Or we add a replace all for `&lt;br&gt;` to `<br>`...

Reply:
nice idea